src/EnergyPlus/HVACInterfaceManager.hh

```diff
@@ -65,6 +65,7 @@ struct EnergyPlusData;
 namespace HVACInterfaceManager {
 
     // Common Pipe Recirc Flow Directions
+    // can't change these to enum class since these are used in SetupOutputVariable()
```

Comment:
Agreed.  I think the root of the problem is that we have given the values of these discrete variables meaning by exposing them to the time series output.  I think it would've been better to just report the flow rates as a positive or negative value so that we wouldn't be reporting discrete terms.  I dunno, just thinking.